Dockerfile

```diff
@@ -60,22 +59,26 @@ RUN apt-get update && \
     libsasl2-dev \
     unzip \
     libsasl2-modules-gssapi-mit && \
-  # MSSQL ODBC Driver:
-  curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - && \
-  curl https://packages.microsoft.com/config/debian/10/prod.list > /etc/apt/sources.list.d/mssql-release.list && \
-  apt-get update && \
-  ACCEPT_EULA=Y apt-get install -y msodbcsql17 && \
-  apt-get clean && \
-  rm -rf /var/lib/apt/lists/*
+    apt-get clean && \
+     rm -rf /var/lib/apt/lists/*
 
+
+ARG TARGETPLATFORM
 ARG databricks_odbc_driver_url=https://databricks.com/wp-content/uploads/2.6.10.1010-2/SimbaSparkODBC-2.6.10.1010-2-Debian-64bit.zip
-RUN wget --quiet $databricks_odbc_driver_url -O /tmp/simba_odbc.zip \
-  && chmod 600 /tmp/simba_odbc.zip \
-  && unzip /tmp/simba_odbc.zip -d /tmp/ \
-  && dpkg -i /tmp/SimbaSparkODBC-*/*.deb \
-  && echo "[Simba]\nDriver = /opt/simba/spark/lib/64/libsparkodbc_sb64.so" >> /etc/odbcinst.ini \
-  && rm /tmp/simba_odbc.zip \
-  && rm -rf /tmp/SimbaSparkODBC*
+RUN if [ "$TARGETPLATFORM" = "linux/amd64" ]; then \
+    curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - \
+    && curl https://packages.microsoft.com/config/debian/10/prod.list > /etc/apt/sources.list.d/mssql-release.list \
+    && apt-get update \
+    && ACCEPT_EULA=Y apt-get install  -y --no-install-recommends msodbcsql17 \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/* \
+    && curl "$databricks_odbc_driver_url" --output /tmp/simba_odbc.zip \
+    && chmod 600 /tmp/simba_odbc.zip \
+    && unzip /tmp/simba_odbc.zip -d /tmp/ \
+    && dpkg -i /tmp/SimbaSparkODBC-*/*.deb \
+    && printf "[Simba]\nDriver = /opt/simba/spark/lib/64/libsparkodbc_sb64.so" >> /etc/odbcinst.ini \
+    && rm /tmp/simba_odbc.zip \
+    && rm -rf /tmp/SimbaSparkODBC*; fi
```

Comment:
Did you have issues with it? Because I believe installing the Databricks ODBC driver should work on ARM.

Reply:
Why would an amd64 library file work on ARM?

Reply:
🤦 oops. You're right. It won't block the build, but won't work.